scripts/deploy/nlp/deploy_triton.py

```diff
@@ -73,6 +74,9 @@ def get_args(argv):
     parser.add_argument(
         "-tha", "--triton_http_address", default="0.0.0.0", type=str, help="HTTP address for the Triton server"
     )
+    parser.add_argument(
+        "-trt", "--triton_request_timeout", default=60, type=int, help="Timeout in seconds for Triton server"
+    )
```

Comment:
Moved the args  `triton_request_timeout` from `nemo/service/deploy/config.json` to the argparser of `deploy_triton.py` so that users can pass it. Also added `openai_format_response` as an user defined option while running `deploy_triton.py` with REST service True

Reply:
Also deleted `config.json` as triton IP, triton port, triton timeout args are passed by the user while running `deploy_triton.py` and they are stored in a json file when `deploy_triton.py` is run which are then accessed in `rest_model_api.py` via the stored config.json file.